VERSIONS.md

```diff
@@ -17,6 +17,10 @@ the License.
 Note that the LinkedIn build system occasionally requires that we skip a
 version bump, so you will see a few skipped version numbers in the list below.
 
+0.9.0
+
+* LIHADOOP-18499 Add extra typed methods for Hadoop DSL SparkJob properties
```

Comment:
Can you bump to 0.9.1 here and also in the gradle.properties file? I published version 0.9.0 already, but I forgot to update the VERSION.md file here.